src/handle_commit.cxx

```diff
@@ -460,30 +465,43 @@ bool raft_server::apply_config_log_entry(ptr<log_entry>& le,
     return true;
 }
 
-void raft_server::snapshot_and_compact(ulong committed_idx) {
+bool raft_server::create_snapshot() {
```

Comment:
@alesapin we can use this to create snapshot on exit

Reply:
Yeah!